src/docs/product/sentry-basics/search/saved-searches.mdx

```diff
@@ -2,72 +2,51 @@
 title: Saved Searches
 sidebar_order: 20
 keywords: ["saved search", "premade searches"]
-description: "Learn more about recommended, pinned, and saved searches."
+description: "Learn more about default, recommended, and saved searches."
 ---
 
-In the "Saved Searches" tab of **Issues**, you can access recommended searches, pin searches that you find most useful, and take advantage of organization-wide saved searches.
+Sentry allows you to save searches you've made on the **Issues** page, either for yourself or for your entire organization. You can access these in the "Saved Searches" sidebar by clicking the "Saved Searches" button at the top right of the page.
 
-## Recommended Searches
-
-Recommended searches are common search terms that we think you're likely to use. These premade searches are listed under "Recommended Searches" in the "Saved Searches" dropdown, and they're listed in order of when you've most recently used them.
-
-![Open Saved Searches dropdown on the Issues page.](search-premade-dropdown.png)
-
-## Pinned Searches
-
-You can pin a search, and it will become the default view you see on the **Issues** page. The pinned search is visible only to you and is relevant across your projects.
-
-1. Type search terms into the search bar.
-
-   ![The search bar of the Issues page with two search terms.](search-pinning-step-one.png)
+![Saved searches button located in the issues page.](search-saved-searches-button.png)
 
-2. Click the pin icon next to that search.
-
-   ![Cursor mousing over the pin icon and tool-tip displayed.](search-pinning-step-two.png)
-
-3. Once pinned, Sentry will add the search to the "Saved Searches" dropdown. The search label in the text will read, "My Pinned Search".
-
-   ![Pinned search in Saved Searches dropdown.](search-pinning-step-three.png)
-
-### Changing a Pinned Search
-
-To change your pinned search:
+## Recommended Searches
 
-1. Select your pinned search. Un-click the pin icon. Your default search will return to `is:unresolved`.
+Recommended searches are common search terms that we think you're likely to use. These premade searches are listed under "Recommended Searches" in the "Saved Searches" sidebar.
 
-2. Run another search. Click the pin icon. The query listed as "My Pinned Search" will now be the new pinned query, replacing the original one.
+![Recommended searches section in the saved searches sidebar.](search-recommended.png)
 
-### Pinning a Recommended Search
+## Custom Saved Searches
 
-You can pin a recommended search the same way you pin any other search. When you've selected a recommended search, and the recommended search query populates the search bar, pin it.
+You can create your own saved searches, which you can make visible to just you, or to your entire organization (if you are an organization owner or manager). These saved searches are not associated with a specific project, but with all projects across the organization.
```

Comment:
I want to be more explicit about the fact that we're talking to two different sets of "you" here--members who are owners/managers and members who aren't. If you're anything besides an owner/manager, your saved searches are only visible to you and can't be made visible to anyone else so let's be really clear about that.

```suggestion
Organization members in all roles can create their own saved searches. These searches are only visible to the person who created them and apply to all their projects. If you're an organization owner or manager, you have the option to make your saved searches visible only to yourself or to your entire organization. Organization-wide saved searches are not associated with a specific project, but with all projects across the organization.
```

Reply:
Agree, big improvement here! Much harder to misinterpret